core/test/fixtures/user-flows/reports/sample-flow-result.json

```diff
@@ -18550,7 +18550,7 @@
             "numericUnit": "millisecond",
             "displayValue": "0.2 s",
             "metricSavings": {
-              "TBT": 25.85177364864864
+              "TBT": 50
```

Comment:
Gah, I can see the appeal of using `Math.floor()` in examples like this. Because it's potential _savings_ we don't want to overestimate the value. Agreed with @connorjclark that rounding 199 to 100 isn't great, but so is overpromising the savings by 2x here.

OTOH using a precision of 50 and `Math.round()` means the rounded value is only ever a max of 25ms away from the original value. To do the same with `Math.floor()` we'd have to drop the granularity to 25, which may be getting too precise again.

IMO in terms of purity (don't overpromise savings you won't be able to get) `Math.floor()` makes the most sense. However in practice, considering the chosen granularity levels and the lack of actual accuracy in the initial estimate, I think it probably doesn't matter, and so `Math.round()` works fine and arguably better.

Reply:
Yeah, I think the case I'm more worried about is this audit score goes from 0.5 to 1 just because of 25.x ms of savings. Perhaps a follow-up PR could introduce some minimum threshold for that condition as well.